src/runtime_shared.js

```diff
@@ -13,7 +13,7 @@
     if (!shouldExport) {
       if (MODULARIZE && EXPORT_ALL) {
         shouldExport = true;
-      } else if (AUDIO_WORKLET && (x == 'HEAP32' || x == 'HEAPU32')) {
+      } else if (AUDIO_WORKLET && (x == 'HEAP32' || x == 'HEAPU32' || x == 'HEAPF32')) {
```

Comment:
Does this mean that if the code in the audio worklet accesses any other heaps that these 3 it would fail?

Reply:
> Does this mean that if the code in the audio worklet accesses any other heaps that these 3 it would fail?

I don't think so - it is only for the scope of audio_worklet.js wrapper, which needs these explicitly exported. Other code works like usual, accessing the heaps via the usual JS code inside the module.

Reply:
So I'm not sure why this change is needed then?  (since there is no corresponding change in `audio_worklet.js`).  What am I missing?

Reply:
Without this change the worklet fails to build with minimal runtime. The tests weren’t catching it (same as the Closure tests weren’t running).

Reply:
> So I'm not sure why this change is needed then?

To make e.g. this line work: https://github.com/emscripten-core/emscripten/blob/d9c260d877fab10f34241325ec4c4c2b072d899c/src/audio_worklet.js#L70

Reply:
I see.  The `globalThis.HEAPF32 = Module['HEAPF32'];` line already existed but the heap wasn't always exported.  Shouldn't the list of heaps here exactly match the list of heaps set of `globalObject` in `audio_worklet.js`?

i.e. can we remove `HEAP32` from the list here?

Reply:
> i.e. can we remove `HEAP32` from the list here?

Isn’t it required by `stackAlloc`?

Reply:
> i.e. can we remove HEAP32 from the list here?

Leave for a separate PR.

Reply:
> > i.e. can we remove HEAP32 from the list here?
> 
> Plausible. Leave for a separate PR.

Since I needed to re-run the (erroneous) failed tests I took the excuse to remove `HEAP32`. Testing manually beforehand its removal runs fine with `MINIMAL_RUNTIME`.